docs/develop/requirements-property-unified-manifest.md

```diff
@@ -0,0 +1,203 @@
+                        }
+                    ]
+                }
+            }
+        ]
+    }
+]
+```
+
+### extensions.autoRunEvents.requirements
+
+The "extensions.autoRunEvents" property configures an add-in to run specified code automatically in response to specified events. The "requirements" subproperty can be used to block this behavior in some versions of Office.
+
+For example, suppose an Outlook add-in is configured to autolaunch in response to the **OnMailSend** event and suppose that the code in the function that runs requires the **Mailbox 1.13** requirement set. But the add-in has other features that would be useful Office versions that only support version 1.12. To ensure that the add-in is installable on versions that support 1.12, a developer can set the “extensions.requirements.capabilities” property to the requirement set **MailBox 1.12** instead of 1.13. But to block the autolaunch feature in versions that don't support 1.13, the developer can add an “extensions.autoRunEvents.requirements.capabilities” property that specifies **Mailbox 1.13**. The following is an example.
+
+```json
+"extensions": [
+    ...
+    {
+        ...
+        "autoRunEvents": [
+            ...
+            {
+                ...
+                "events": {
+                    "type": "OnMailSend",
+                    "actionId": "logOutgoingEmail",
+                    "options": {
+                        "sendMode": "promptUser"
+                    }
+                },
+                "requirements": {
+                    "capabilities": [
+                        {
+                            "name": "MailBox",
+                            "minVersion": "1.13"
+                        }
+                    ]
+                }
+            }
+        ]
+    }
+]
+```
+
+### extensions.ribbons.requirements
+
+The "extensions.ribbons" property is used to customize the Office application ribbon when the add-in is installed. The "requirements" subproperty can be used to prevent the customizations in some versions of Office.
+
+For example, suppose an Outlook add-in is configured to add a custom button to the ribbon and the button runs a function that uses code introduced in the **Mailbox 1.9** requirement set. But the add-in has other features that would be useful on versions of Office that only support version 1.8. To ensure that the add-in is installable on versions that support 1.8, a developer can set the “extensions.requirements.capabilities” property to the requirement set **MailBox 1.8** instead of 1.9. But to block the custom button from appearing on the ribbon in versions that don't support 1.9, the developer can add an “extensions.ribbons.requirements.capabilities” property that specifies **Mailbox 1.9**. The following is an example. For details of the custom ribbon configuration, see [Create add-in commands with the unified manifest for Microsoft 365](create-addin-commands-unified-manifest.md).
```

Comment:
```suggestion
For example, suppose an Outlook add-in is configured to add a custom button to the ribbon and the button runs a function that uses code introduced in the **Mailbox 1.9** requirement set. But the add-in has other features that would be useful on versions of Office that only support version 1.8. To ensure that the add-in is installable on versions that support 1.8, a developer can set the "extensions.requirements.capabilities" property to the requirement set **MailBox 1.8** instead of 1.9. But to block the custom button from appearing on the ribbon in versions that don't support 1.9, the developer can add an "extensions.ribbons.requirements.capabilities" property that specifies **Mailbox 1.9**. The following is an example. For details of the custom ribbon configuration, see [Create add-in commands with the unified manifest for Microsoft 365](create-addin-commands-unified-manifest.md).
```